src/test/groovy/life/qbic/datamodel/dtos/business/CustomerSpec.groovy

```diff
@@ -0,0 +1,21 @@
+package life.qbic.datamodel.dtos.business
+
+import spock.lang.Specification
+
+/**
+ * Simple tests for the customer dto class
+ *
+ * @author Sven Fillinger
+ * @since 1.11.0
+ */
+class CustomerSpec extends Specification{
+
+  def "wrong email shall raisa a validation exception"() {
```

Comment:
```suggestion
  def "wrong email shall raise a validation exception"() {
```

Reply:
that is slang: "rais 'a xeption."

Reply:
I didn't know that Jar Jar Binks is working at QBiC :D